webapp/pages/index.vue

```diff
@@ -151,7 +151,7 @@ export default {
           orderBy: this.sorting,
         },
         updateQuery: (previousResult, { fetchMoreResult }) => {
-          if (!fetchMoreResult || fetchMoreResult.Post.length < this.pageSize) {
+          if (!fetchMoreResult || fetchMoreResult.Post.length <= this.pageSize) {
```

Comment:
I disagree. Most of the time, if there is more stuff to fetch, the length of `fetchMoreResult` is the same as `pageSize`. We have the edge case, that if your last page has the same number of items as `pageSize`, the load button is there when it shouldn't.

Reply:
yeah, was a bit off on this logic hahah